chunkey/encode_pipeline.py

```diff
@@ -401,19 +388,17 @@ def _upload_transport(self):
                         self.video_id,
                         transport_stream
                     ))
-                """
-                Actually upload the thing
-                """
+                    # Actually upload the thing
                 sys.stdout.write('\r')
-                sys.stdout.write("%s : %s" % ('Upload', transport_stream))
+                sys.stdout.write("%s : %s" % ('Upload', transport_stream))  # pylint: disable=unicode-format-string
                 upload_key.set_contents_from_filename(
                     os.path.join(self.video_root, transport_stream)
                 )
                 upload_key.set_acl('public-read')
                 sys.stdout.flush()
 
         if self.settings.DELIVER_ROOT is not None and \
-                len(self.settings.DELIVER_ROOT) > 0:
+                self.settings.DELIVER_ROOT:
```

Comment:
I think you changed the meaning here...the second part of the `if` was trying to catch the case where DELIVER_ROOT is set, but set to an empty string.  You could rewrite this as:
```
if self.settings.DELIVER_ROOT and self.settings.DELIVER_ROOT.strip():
```

Reply:
Could you explain you suggested rewrite please? I'm having a hard time seeing how it mimics what was written beforehand. Specifically, why are you using the strip() function?